include/TrustWalletCore/TWZilliqaSigner.h

```diff
@@ -12,8 +12,6 @@
 
 TW_EXTERN_C_BEGIN
 
-const uint32_t TWZilliqaTxVersion = 65537;
```

Comment:
Why not keep the constant, and move its value to CPP file to avoid duplicate symbol?  Also should be static.

Reply:
My first try is `extern const` and move this value to cpp, but we tend to fetch network id / chain id from rpc, Android doesn't use this variable either (need JNI)

Reply:
This value only used in test, so it's safe to remove it